app/src/main/res/menu/main_menu.xml

```diff
@@ -6,7 +6,7 @@
     <item android:id="@+id/action_search"
           android:title="@string/action_search"
           android:icon="@drawable/ic_search_white_24dp"
-          pwstore:showAsAction="ifRoom|collapseActionView"
+          pwstore:showAsAction="always|collapseActionView"
```

Comment:
Leaving this on "ifRoom" causes the search icon to disappear after the search bar is initially closed.
My best guess is that the measurement that gets used to determine if there is room happens after the bar is expanded, since forcing a redraw seems to fix the problem (e.g. rotating the screen, etc).

In any case I don't think this is too big of a deal since there's only ever two buttons on the action bar anyways, I don't think we'll run into layout issues.

Reply:
I'm fine with it this way.